tensorboard/plugins/mesh_visualizer/BUILD

```diff
@@ -0,0 +1,116 @@
+package(default_visibility = ["//tensorboard:internal"])
+
+licenses(["notice"])  # Apache 2.0
+exports_files(["LICENSE"])
+
+load("//tensorboard/defs:protos.bzl", "tb_proto_library")
+
+py_library(
+    name = "metadata",
+    srcs = ["metadata.py"],
+    srcs_version = "PY2AND3",
+    deps = [
+        ":protos_all_py_pb2",
+        "//tensorboard/compat/proto:protos_all_py_pb2",
+    ],
+)
+
+py_test(
+    name = "metadata_test",
+    size = "small",
+    srcs = ["metadata_test.py"],
+    srcs_version = "PY2AND3",
+    deps = [
+        ":metadata",        
+        "//tensorboard:expect_tensorflow_installed",
+        "//tensorboard/util:test_util",
+        "@org_pythonhosted_six",
+    ],
+)
+
+py_library(
+    name = "mesh_plugin",
+    srcs = ["mesh_plugin.py"],
+    srcs_version = "PY2AND3",
+    visibility = ["//visibility:public"],
+    deps = [
+        ":metadata",
+        ":protos_all_py_pb2",
+        "//tensorboard:expect_numpy_installed",        
+        "//tensorboard:plugin_util",
+        "//tensorboard/backend:http_util",
+        "//tensorboard/backend/event_processing:event_accumulator",        
+        "//tensorboard/plugins:base_plugin",
+        "//tensorboard/util:tensor_util",
+        "@org_pythonhosted_six",
+        "@org_pocoo_werkzeug",
+    ],
+)
+
+py_library(
+    name = "test_utils",
+    testonly = 1,
+    srcs = ["test_utils.py"],
+    srcs_version = "PY2AND3",
+    deps = [
+        ":mesh_summary",
+        "//tensorboard:expect_tensorflow_installed",        
+        "//tensorboard/backend:application",
+        "//tensorboard/backend/event_processing:event_multiplexer",
+        "@org_pocoo_werkzeug",
+    ],
+)
+
+py_test(
+    name = "mesh_plugin_test",
+    size = "small",
+    srcs = ["mesh_plugin_test.py"],
+    srcs_version = "PY2AND3",
+    deps = [
+        ":mesh_plugin",
+        ":mesh_summary",
+        ":test_utils",
+        "//tensorboard:expect_numpy_installed",        
+        "//tensorboard:expect_tensorflow_installed",        
+        "//tensorboard/backend:application",
+        "//tensorboard/backend/event_processing:event_multiplexer",
+        "//tensorboard/plugins:base_plugin",
+        "//tensorboard/util:test_util",
+        "@org_pocoo_werkzeug",
+        "@org_pythonhosted_six",
+        "@org_pythonhosted_mock",
+    ],
+)
+
+py_library(
+    name = "mesh_summary",
+    srcs = ["mesh_summary.py"],
```

Comment:
All other plugins call this file `summary.py` and the target `:summary`.
Could you please retain that convention?